src/ProjectSystem/Package/Microsoft.NET.Sdk.Web.ProjectSystem.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <IsPackable>true</IsPackable>
+    <IsShipping>false</IsShipping>
```

Comment:
Should this be IsShippingPackage? https://github.com/dotnet/sdk/blob/87dcc1d8658e25d23e49ab4573044b95668bf734/Directory.Build.props#L15

I recall there was a whole debate for SDK where IsShipping=false turned off too much stuff like localization

Reply:
Good thinking. You think that for each and every one of these?

Reply:
Yes. These are all things that ship in C:\Program Files\dotnet\sdk\(version)\Sdks, the package is just to transport it to there, but the bits inside the package ship to customers.